frame/dvm/ethereum/src/account_basic.rs

```diff
@@ -118,52 +118,63 @@ where
 	C: Currency<T::AccountId>,
 {
 	/// Get the account basic in EVM format.
-	fn account_basic(address: &H160) -> EVMAccount {
-		let account_id = <T as darwinia_evm::Config>::IntoAccountId::into_account_id(*address);
+	fn account_basic(addr: AddrWrapper<T::AccountId>) -> EVMAccount {
+		let account_id = match addr {
+			AddrWrapper::ETH(addr) => {
+				<T as darwinia_evm::Config>::IntoAccountId::into_account_id(addr)
+			}
+			AddrWrapper::Substrate(account_id) => account_id,
+		};
```

Comment:
Prefer to add two method if required here instead of match switch here. Let's keep it simple.